test/torch/test_hook.py

```diff
@@ -220,3 +220,69 @@ def test_hook_args_and_cmd_signature_malleability():
 
     r3 = a + b
     assert (r3 == syft.LoggingTensor().on(torch.tensor([2.0, 4]))).all()
+
+
+def test_RNN_grad_set_backpropagation():
+    """Perform backpropagation at a remote worker and check if the gradient updates
+    and properly computed within the model"""
+
+    import syft as sy
+
+    hook = sy.TorchHook(torch)
+    alice = sy.VirtualWorker(hook, id="alice")
```

Comment:
You can get Alice by including workers as a parameter and writing Alice = workers["Alice"]. The tests for most tensors have good examples for how to do this

Reply:
Done!